specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-01-02-preview/managedClusters.json

```diff
@@ -9399,7 +9425,7 @@
       "required": true,
       "type": "string",
       "description": "The name of trusted access role binding.",
-      "pattern": "^([A-Za-z0-9-])+$",
+      "pattern": "^([A-Za-z0-9-])$",
```

Comment:
Why changing the pattern of `TrustedAccessRoleBindingNameParameter`? Seems not related to your feature.

Reply:
whoops that was a mistake. fixed now.